RELEASE-NOTES.md

```diff
@@ -1,152 +1,153 @@
 # Release Notes
 
-<!--
-⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠
-Do not create individual sections for the beta releases.
-Instead update the vNext section until 4.0.0 is out.
-⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠
--->
+## PyMC 4.0.0
+⚠️ The changes below are the delta between the releases `v3.11.5` →...→ `v4.0.0`.
 
-## PyMC vNext (4.0.0b1 → ... → 4.0.0b6 → 4.0.0)
-⚠ The changes below are the delta between the releases `v3.11.5` →...→ `v4.0.0`.
+We the people of PyMC know that you, our dear users, are curious folks (yes, we see you), so **we also put together a [release announcement](https://www.pymc.io/blog/v4_announcement.html) on our website**.
+Feel free to read it, print it out, and give it to people on the street -- because _everybody_ has to know PyMC 4.0 is officially out 🍾
 
-### Not-yet working features
-We plan to get these working again, but at this point their inner workings have not been refactored.
-- Timeseries distributions (see [#4642](https://github.com/pymc-devs/pymc/issues/4642))
-- Nested Mixture distributions (see [#5533](https://github.com/pymc-devs/pymc/issues/5533))
-- `pm.sample_posterior_predictive_w` (see [#4807](https://github.com/pymc-devs/pymc/issues/4807))
-- Partially observed Multivariate distributions (see [#5260](https://github.com/pymc-devs/pymc/issues/5260))
+### Do not miss 🚨
 
-Also check out the [milestones](https://github.com/pymc-devs/pymc/milestones) for a potentially more complete list.
+  - ⚠️ The project was renamed to "PyMC". Now the library is installed as "pip install pymc" and imported like `import pymc as pm`. See this [migration guide](https://www.pymc-labs.io/blog-posts/the-quickest-migration-guide-ever-from-pymc3-to-pymc-v40/) for more details.
+  - ⚠️ Theano-PyMC has been replaced with Aesara, so all external references to `theano` and `tt` need to be replaced with `aesara` and `at` (see [4471](https://github.com/pymc-devs/pymc/pull/4471)).
+  - ⚠️ Random seeding behavior changed (see [#5787](https://github.com/pymc-devs/pymc/pull/5787))!
```

Comment:
I think the global seeding not having an effect is really important. Publication code is likely to use seeding for reproducible results so users should be aware the old global seeding approach (which we use to use in all examples) won't work anymore. 

The fact that it changed is less important but might as well mention everything together for easier reading.

Reply:
Random seed setting is indeed something that users pay attention to -- we often get this question when teaching PyMC ("how do I make sure my sampling results are reproducible?"). So in that regard I agree with @ricardoV94 

> The fact that it changed is less important but might as well mention everything together for easier reading.

Agreed. As it's not too long, having everything here makes it easier to navigate for the reader, instead of having the random seed setting info scattered throughout the notes

Reply:
👍